### PR TITLE
Part 1: Database scheme and dummy data

### PR DESCRIPTION
**Pull request by benjimarshall**

### CODE REVIEW

File: SqlQueries/BookishSchema.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    DROP TABLE loans;
+
+IF OBJECT_ID('bookcopies') IS NOT NULL
+    DROP TABLE bookcopies;
+
+IF OBJECT_ID('books') IS NOT NULL
+    DROP TABLE books;
+
+IF OBJECT_ID('users') IS NOT NULL
+    DROP TABLE users;
```
Comment (FridaTveit) [inline SqlQueries/BookishSchema.sql:19]: Why all the drop table statements and commented out lines? 🙂

File: SqlQueries/InsertData.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
+INSERT INTO users(username) VALUES (N'Benji');
+INSERT INTO users(username) VALUES (N'Frank');
+INSERT INTO users(username) VALUES (N'Frida');
+INSERT INTO users(username) VALUES (N'Shiv');
```
Comment (FridaTveit) [inline SqlQueries/InsertData.sql:4]: You can do these in one statement:
```
INSERT INTO users (username) VALUES ('Benji'), ('Frank'), ...;
```
You also shouldn't need the N 🙂

File: SqlQueries/InsertData.sql [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+INSERT INTO bookcopies(isbn) VALUES (N'978-0-12-383872-8');
+INSERT INTO bookcopies(isbn) VALUES (N'978-0-12-383872-8');
+INSERT INTO bookcopies(isbn) VALUES (N'978-0-12-385060-7');
+INSERT INTO bookcopies(isbn) VALUES (N'978-0-262-03293-3');
+INSERT INTO bookcopies(isbn) VALUES (N'978-0-262-03293-3');
+
+INSERT INTO loans(bookid, userid, due) VALUES (1, 1, '2020-07-15');
+INSERT INTO loans(bookid, userid, due) VALUES (5, 1, '2020-11-07');
+INSERT INTO loans(bookid, userid, due) VALUES (2, 2, '2021-04-01');
+INSERT INTO loans(bookid, userid, due) VALUES (6, 4, '2020-10-25');
```
Comment (FridaTveit) [inline SqlQueries/InsertData.sql:21]: To make sure these ids are correct you can actually set them when creating the book copies and users (by using `SET IDENTITY_INSERT bookcopies ON` and same for users) 🙂

**Review by FridaTveit — COMMENTED** [review]
A few questions and suggestions, but overall the structure looks good! 🙂

File: SqlQueries/BookishSchema.sql [hunk 4]
```diff
@@ -0,0 +1,42 @@
+-- Delete and recreate the cat_shelter database
```
[the remark this answers is not in the document]
Comment (benjimarshall) [reply SqlQueries/BookishSchema.sql]: Whoops, that's from the old template we were given... Removed in f85e313 🙂

File: SqlQueries/BookishSchema.sql (hunk 1 above)
Comment (benjimarshall) [reply SqlQueries/BookishSchema.sql:13]: The commented out lines were from the old template and shouldn't have been committed. They've been removed in f85e313 🙂

As for the drop table statements, the template schema we started with had dropped all of the relevant tables, and then remade them. I assumed this was some kind of convention and ran with it, but I can remove them if you like 🙂

File: SqlQueries/InsertData.sql (hunk 3 above)
Comment (benjimarshall) [reply SqlQueries/InsertData.sql]: Thanks for that! Change in 8ad69ed 🙂

File: SqlQueries/InsertData.sql (hunk 2 above)
Comment (benjimarshall) [reply SqlQueries/InsertData.sql]: Thanks for those things! Changed in 8ad69ed 🙂

File: SqlQueries/BookishSchema.sql (hunk 1 above)
Comment (FridaTveit) [reply SqlQueries/BookishSchema.sql:13]: Fair enough, feel free to keep the drop statements. They'll make it easier to run the script again I guess 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**